<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateAccount</class>
 <widget class="QWidget" name="CreateAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>460</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>418</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>4Service</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxBaseInfo">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelFirstName">
        <property name="font">
         <font>
          <family>System</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Имя</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelBirthDate">
        <property name="font">
         <font>
          <family>System</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Дата рождения</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelLastName">
        <property name="font">
         <font>
          <family>System</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Фамилия</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditFirstName"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditLastName"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditBirthDate"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxExtraInfo">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="labelNameCompany">
        <property name="font">
         <font>
          <family>System</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Название организации</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditNameCompany"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelAdress">
        <property name="font">
         <font>
          <family>System</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Адрес</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditAdress"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelLineBuisness">
        <property name="font">
         <font>
          <family>System</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Сфера деятельности</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditLineBuisness"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelAboutCompany">
        <property name="font">
         <font>
          <family>System</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Об организации</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>216</width>
           <height>98</height>
          </rect>
         </property>
         <widget class="QTextEdit" name="textEditAboutCompany">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>221</width>
            <height>101</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBoxLogPass">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="labelLogin">
        <property name="font">
         <font>
          <family>System</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Логин</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditLogin"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelPassword">
        <property name="font">
         <font>
          <family>System</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Пароль</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditPassword"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelPasswordAgain">
        <property name="font">
         <font>
          <family>System</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Повторите пароль</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditCheckPassword"/>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <family>Terminal</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButtonRegistration">
     <property name="font">
      <font>
       <family>System</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Регистрация</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
